tests/test_url_config.py

```diff
@@ -0,0 +1,55 @@
+
+import strax
+import straxen
+import numpy as np
+from warnings import warn
+from straxen.test_utils import nt_test_context, nt_test_run_id
+import unittest
+
+
+class TestPlugin(strax.Plugin):
+    depends_on = ()
+    dtype = strax.time_fields
+    provides = ('test_data',)
+    test_config = straxen.URLConfig(default=42,)
+    
+    def compute(self):
+        pass
+
+
+class TestURLConfig(unittest.TestCase):
```

Comment:
Maybe also add a test for the `protocol_descr` and `fsspec`, mostly because I think these tests are actually super instructive examples of how the code should be used.

Reply:
added a test for fsspec protocol. not sure how to test protocol_descr method since its just a helper method to quickly see what protocols are registered and their call signature